src/pepr/operator/crd/sources/package/v1alpha1.ts

```diff
@@ -303,6 +303,42 @@ const sso = {
           type: "string",
         },
       },
+      protocolMappers: {
+        description: "Protocol Mappers to configure on the client",
+        type: "array",
+        default: [],
```

Comment:
@UnicornChance @mjnagel @ericwyles @ntwkninja I added the additional `default: []` to ensure if you remove the `protocolMappers` key from the Package's sso configuration it will remove any existing protocolMappers from the dedicated scope. 

Alternatively we could not specify a default here and it would not 'delete' any previously created `protocolMappers` unless you explicitly set `protocolMappers: []`.

Reply:
I think this approach makes sense - we should lean into declarative nature here and if protocol mappers get added/removed from Package CR I would expect the same to happen on the keycloak side.